services/server/src/fawry/types/requests.ts

```diff
@@ -5,38 +5,46 @@ import {
 } from "@/fawry/types/ancillaries";
 
 export type GeneralRequestPayload = {
```

Comment:
Let's call it `BaseRequestPayload`. This will indicate that other requests will extend this base type.